notes/2.md

```diff
@@ -68,6 +68,79 @@ Set up scripts
   }
 }
 ```
+Create empty `gatsby-config.js` in www directory
+![image](https://user-images.githubusercontent.com/3941856/74722809-63ffdb80-51ee-11ea-97a3-910159021ac8.png)
+
+```javascript
+module.exports = {}
+```
+Create www/src/pages/index.js
+![image](https://user-images.githubusercontent.com/3941856/74723055-cc4ebd00-51ee-11ea-9655-d3cc51981452.png)
+
+```javascript
+export default props => (
+  <div>
+    <h1>Our Site</h1>
+  </div>
+);
+```
+
+Go back to the root of the project and run
+
+```
+yarn workspace www develop
+```
+
+Get an error, import React in index.js to fix it
+
+```javascript
+import React from "react";
+export default props => (
+  <div>
+    <h1>Our Site</h1>
+  </div>
+);
+```
+
+Add packages directory, package.json, and yarn.lock to git 
+
+```
+ga package* yarn.lock
```

Comment:
I thought this was clever. I understand what the `*` does but I wouldn't have thought to use it this way.